t/numbers.t

```diff
@@ -38,5 +38,24 @@ foreach my $test_ref (@tests) {
 	is(remove_insignificant_digits($input), $expected);
 }
 
+@tests = (
+["1","1"],
+["1.0","1.0"],
+["2,2","2.2"],
+["2,20","2.20"],
+["100 000","100000"],
+["33,000,000","33000000"],
+["3,760,00","3760"],
+);
```

Comment:
```suggestion
["1,000.22","1000.22"],
);
```
both a thousand separator and a decimal at the same time, for completeness?

Reply:
Thanks @svensven , I'll add it.

Reply:
added in https://github.com/openfoodfacts/openfoodfacts-server/pull/4317